src/taipy/core/data/csv.py

```diff
@@ -109,6 +109,7 @@ def path(self):
 
     @path.setter
     def path(self, value):
+        self._path = value
         self.properties[self.__PATH_KEY] = value
```

Comment:
Indeed we have two ways of storing the path: 
- as self._path
- as self.properties[self.__PATH_KEY]
But only one way was updated. Well spotted!

The fix you proposed is to update both of them. Instead, I would only use a single way of storing it (self.properties[self.__PATH_KEY]). I would completely remove self._path.

DOes it make sense to you? Do I miss something here or do we just need one storage?

Reply:
That means we can have in the __init__ method : 

```
path = properties.get(self.__PATH_KEY, properties.get(self.__DEFAULT_PATH_KEY))
if path is None:
    raise MissingRequiredProperty("A default_path property is required in a CSV data node config.")
else:
    properties[self.__PATH_KEY] = path
```

Reply:
In the reload property we could have:

```
def path(self):
        return self.properties[self.__PATH_KEY]
```